m2m-transfers/V3/REPWRITERSPECS/BANNO.M2MTRANSFERS.V3.POW

```diff
@@ -3354,21 +3357,21 @@ END [PROCEDURE]
 
 PROCEDURE CHECKFORINVALIDCHARS
```

Comment:
I think you'll also want to be checking for invalid characters when pulling the saved name from the external account records.

Reply:
Added.